<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1203</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>1151</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ejercicio 1</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>20</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Calificacion de Estudiantes</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="teEstudiantes">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>120</y>
        <width>421</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>70</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese los datos de los estudiantes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>500</y>
        <width>111</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Crear reporte</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>100</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>El formato es: Nombre Calificacion</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ejercicio 2</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>891</width>
        <height>471</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Crear Producto</string>
       </attribute>
       <widget class="QTextEdit" name="teProducto">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>80</y>
          <width>371</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>380</y>
          <width>93</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Crear</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>30</y>
          <width>521</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Ingrese los productos con el formato: &quot;Codigo Nombre Precio Stock&quot;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>380</y>
          <width>161</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Leer archivo existente</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Modificar producto</string>
       </attribute>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>100</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Codigo:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>150</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Stock:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>50</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Modificar producto</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>230</y>
          <width>93</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Modificar</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="sbcodigo">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>100</y>
          <width>61</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="sbstock">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>150</y>
          <width>61</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Ejercicio 3</string>
     </attribute>
     <widget class="QTextEdit" name="teCSV">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>671</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>50</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CSV</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>530</y>
        <width>101</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Generar CSV</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>622</x>
        <y>530</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Generar reporte</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="bton_Salir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1203</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
